### DISCUSSION TITLE
Two Things not yet reflected

### DISCUSSION THREAD

**Discussion by gogog01-29-2021** [mannmi/KoreanWonDetection#16] [Ideas]
1. Function that makes people moves to proposal template if they click submit idea button

2. Philosophy not fully reflected on Github README.md

3. README.md
contributing.md and other files URL are not working

**Comment by mannmi** [comment]
1) The template has been implemented for bug but not yet for idear proposals this will be remedied asap :)

2) Please elaborate on the infromation which has to be added to the Philosophy :)

3) Links have been added to bug issue #17. This is a bug please create a issue when you encounter bugs.

**Comment by mannmi** [comment]
Hello gogog01-29-2021

issue #17 has been resolved please check if it meets your requirements, in case it should not open a new issue with a detailed description of which links are broken. Thank you in advanced.

Kind Regards,
mannmi

**Comment by mannmi** [comment]
I think we can close this discusion at this stage :) please reopen again if i missed sommthin.
This thread is marked as "stale".

:) thread will be closed if no further activity is detected.
